packages/babel-plugin-proposal-object-rest-spread/test/fixtures/object-rest/template-literal-allLiterals-true-no-hoisting/input.js

```diff
@@ -0,0 +1,9 @@
+const example = () => {
+  const input = {};
+  const foo = 'foo';
+
+  const {
+    [`${foo}_bar`]: country,
+    ...rest
+  } = input;
```

Comment:
This is an object destructure. We need to test an object rest, which is (confusingly) when there's no `const` declaration:
```suggestion
  ({
    [`${foo}_bar`]: country,
    ...rest
  } = input);
```

Reply:
Is it? I tought both the desturcturing and `object-rest-spread` plugins transformed both the syntaxes, it just depends on which plugin you enabled.

This test is currently broken on `main` when targeting `chrome 55` (which supports destructuring, but not object rest/spread):
https://babeljs.io/repl/build/main#?browsers=chrome%2055&build=&builtIns=false&corejs=3.6&spec=false&loose=false&code_lz=MYewdgzgLgBApgDwIYFsAOAbOMC8MAUAlLgHwwDeAUDDKJLAJZhoCuse5AvgNzW3jQYAMxAhcMAOQiQE3nzqCqNGgG0ABgBJy0zgH0ARkgBOagLoAufizBQjATwA0fGgDo3RuND6dxTVlF5OIA&presets=env

Reply:
I am so confused by the split between the two plugins…

Reply:
We should definitely consider merging them, relying on the `targets` top-level option to decide how much to compile on the plugin.